user_tools/pyproject.toml

```diff
@@ -41,7 +41,7 @@ dependencies = [
     "azure-storage-blob==12.17.0",
     "adlfs==2023.4.0"
 ]
-dynamic=["entry-points", "version"]
+dynamic=["version"]
```

Comment:
Could you explain why we need this change?

Reply:
This is an unintended change. I will revert this. Thanks!